charts/rasa-x/values.yaml

```diff
@@ -394,7 +408,7 @@ eventService:
 # app (custom action server) specific settings
 app:
   # default is to install action server from image.
-  install: true
+  install: false
```

Comment:
Why'd' this change?

Reply:
Hmmm, I think after testing I forgot to change it back.